_includes/tutorials/joining-stream-stream/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,25 @@
+CREATE STREAM ORDERS (ORDER_ID INT, ORDER_TS VARCHAR, TOTAL_AMOUNT DOUBLE, CUSTOMER_NAME VARCHAR)
+    WITH (KAFKA_TOPIC='_orders',
+          VALUE_FORMAT='JSON',
+          TIMESTAMP='ORDER_TS',
+          TIMESTAMP_FORMAT='yyyy-MM-dd''T''HH:mm:ssX',
+          PARTITIONS=4);
+
+CREATE STREAM SHIPMENTS (SHIPMENT_ID VARCHAR, SHIP_TS VARCHAR, ORDER_ID INT, WAREHOUSE VARCHAR)
+    WITH (KAFKA_TOPIC='_shipments',
+          VALUE_FORMAT='JSON',
+          TIMESTAMP='SHIP_TS',
+          TIMESTAMP_FORMAT='yyyy-MM-dd''T''HH:mm:ssX',
+          PARTITIONS=4);
```

Comment:
We should call out somewhere that these `4` partitions have significance, and that the streams need to be co-partitioned. We should say simply exactly what that means and what happens if you don't do it. Maybe call out that this happens automatically if it's an operation that happens as a result of other streaming operations.